[sqlite] add a way to provide custom directory for the database (#31278)
expo/expo: docs/pages/versions/unversioned/sdk/sqlite.mdx

}
```

### Sharing a database between apps/extensions (iOS)

To share a database with other apps/extensions in the same App Group you can use shared containers. Follow these steps:
<Step label="1">
Configure the App Group in `app.json`:
```json app.json
{
  "expo": {
    "ios": {
      "bundleIdentifier": "com.myapp",
      "entitlements": {
        "com.apple.security.application-groups": [
          "group.com.myapp"
        ]
      }
    }
  }
}
```
</Step>
<Step label="2">

> Use [`getSharedContainer`](filesystem-next/#getsharedcontainerappgroup) from the [`expo-file-system`](filesystem-next/) module to retrieve the path to the shared container:

Reviewer: ```suggestion
Use [`appleSharedContainers`](filesystem-next/#applesharedcontainers) from the [`expo-file-system`](filesystem-next/) module to retrieve the path to the shared container:
```

```tsx Using Shared Container for SQLite Database on iOS
import { SQLiteProvider, defaultDatabaseDirectory } from 'expo-sqlite';

> import { Directory } from 'expo-file-system/next';

Reviewer: ```suggestion
import { Paths } from 'expo-file-system/next';
```

import { useMemo } from 'react';
import { Platform, View } from 'react-native';

export default function App() {
  const dbDirectory = useMemo(() => {
    if (Platform.OS === 'ios') {

>       return Directory.getSharedContainer('group.com.myapp')?.path;

Reviewer: ```suggestion
      return Object.values(Paths.appleSharedContainers)?.[0]?.uri;
```

}
```

### Sharing a database between apps/extensions (iOS)

> To share a database with other apps/extensions in the same App Group you can use shared containers. Follow these steps:

Reviewer 2: ```suggestion
To share a database with other apps/extensions in the same App Group, you can use shared containers by following the steps below:
```

Update this to avoid using a fragmented second sentence so the readability is improved.

> <Step label="1">
> Configure the App Group in `app.json`:
> ```json app.json

Reviewer 2: ````suggestion
<Step label="1">

Configure the App Group in app config:

```json app.json
````

Nit: Using line breaks like the above here helps us to avoid any unknown markup rendering issues.
Let's change `app.json` to app config since this file can have different file formats in a project (example: app.config.js).

{
  "expo": {
    "ios": {
      "bundleIdentifier": "com.myapp",
      "entitlements": {
        "com.apple.security.application-groups": [
          "group.com.myapp"
        ]
      }
    }
  }
}
```

> </Step>
> <Step label="2">
> Use [`Paths.appleSharedContainers`](filesystem-next/#applesharedcontainers) from the [`expo-file-system`](filesystem-next/) module to retrieve the path to the shared container:

Reviewer 2: ```suggestion

</Step>

<Step label="2">

Use [`Paths.appleSharedContainers`](filesystem-next/#applesharedcontainers) from the [`expo-file-system`](filesystem-next/) library to retrieve the path to the shared container:
```

We prefer using ["library" over module](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#glossary) as per our writing style guide
